project/Dependencies.scala

```diff
@@ -14,8 +15,13 @@ object Dependencies {
 
   val ClusterHttp = Seq(
     libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-cluster" % AkkaVersion,
-      "com.typesafe.akka" %% "akka-http"    % AkkaHttpVersion
+      "com.typesafe.akka" %% "akka-cluster"                      % AkkaVersion,
+      "com.typesafe.akka" %% "akka-http"                         % AkkaHttpVersion,
+      "com.typesafe.akka" %% "akka-http-spray-json-experimental" % "2.4.11",
```

Comment:
I am unsure about this, but I could not find akka-http-spray-json-experimental with version AkkaHttpVersion